unstructured/nlp/tokenize.py

```diff
@@ -114,10 +114,10 @@ def _download_nltk_packages_if_not_present():
 
     tagger_available = check_for_nltk_package(
         package_category="taggers",
-        package_name="averaged_perceptron_tagger_eng",
+        package_name="averaged_perceptron_tagger",
     )
     tokenizer_available = check_for_nltk_package(
-        package_category="tokenizers", package_name="punkt_tab"
+        package_category="tokenizers", package_name="punkt"
     )
 
     if not (tokenizer_available and tagger_available):
```

Comment:
Hi,

The condition on line 120 should be 'or', otherwise it fails to download package when one of the taggers or tokenizers is missing,
    `if not (tokenizer_available or tagger_available):`